### DISCUSSION TITLE
Is it possible to configure consult to allow fuzzy searching for things using the capitals?

### DISCUSSION THREAD

**Discussion by ldeck** [minad/consult#986] [Q&A] [answered]
For example, say I have the files (among others)
- `src/main/java/my/package/SomeJavaFactory.java` 
- - `src/test/java/my/package/SomeJavaFactoryTest.java` 

Searching for the files using upper case letters works by using spaces like "S J Fact".

Is it possible to configure consult to be able to fuzzy search for files such that the spaces for prefixes are optional?

**Comment by minad** [comment, marked as answer]
You probably use `consult-fd` or `consult-find`? There we don't have an easy way to enable fuzzy search for external commands but you can a custom regexp compiler, see `consult--regexp-compiler`.

If you talk about fuzzy filtering for arbitrary commands, take a look at the `flex` completion style or `orderless-flex`.
